snowplow-demo-app/src/main/java/com/snowplowanalytics/snowplowtrackerdemo/ErrorLogger.java

```diff
@@ -0,0 +1,14 @@
+package com.snowplowanalytics.snowplowtrackerdemo;
+
+import android.util.Log;
+
+import com.snowplowanalytics.snowplow.tracker.ErrorLogging;
+
+public class ErrorLogger implements ErrorLogging {
+    @Override
+    public void log(String source, String errorMessage) {
+        if (source.equals("Tracker") || source.equals("Session") || source.equals("ProcessObserver")) {
+            Log.e("Error reported >>> " + source, errorMessage);
```

Comment:
Maybe worth adding a Tracking example here, send the error message into the collector? Just a thought but not a blocker.